cmd/api/src/daemons/datapipe/agi.go

```diff
@@ -155,7 +155,7 @@ func ParallelTagAzureTierZero(ctx context.Context, db graph.Database) error {
 	return nil
 }
 
-func TagActiveDirectoryTierZero(ctx context.Context, db database.Database, graphDB graph.Database) error {
+func TagActiveDirectoryTierZero(ctx context.Context, db appcfg.GetFlagByKeyer, graphDB graph.Database) error {
```

Comment:
We should probably rename this from a `db` var to something like `flag` to make it clear it's not a DB instance. Minor nit, it could ship as is just as easily.

Reply:
Updated!